package.json

```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/Mobiliteitsfabriek/eslint-config-mig-react-native.git"
   },
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
I don't think we should update the version number in a proposal PR:

- Doing so makes it harder to bundle changes into a single release.
- We should make the release structure a bit more formal, for examply by adding a CHANGELOG.md, since this package is released publicly.
- Proposals could be open for a while when thorough discussion or research is required. We can't know the order in which they will be merged in.

Reply:
Added changelog and reverted version bump